data/data/bootstrap/baremetal/files/etc/containers/systemd/image-customization.container

```diff
@@ -28,11 +28,6 @@ Environment="KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback"
 
 [Service]
 EnvironmentFile=/etc/ironic.env
-Environment="NMSTATE_DIR=/tmp/nmstate"
```

Comment:
The same patch should remove the mount on line 15 also, since it depends on this env var (will be remounting `/` without it).

Reply:
Done